rocky/katalogus/client.py

```diff
@@ -58,11 +75,14 @@ def delete_organization(self):
         response = self.session.delete(f"{self.organization_uri}")
         response.raise_for_status()
 
-    def get_all_plugins(self):
-        response = self.session.get(f"{self.organization_uri}/plugins")
-        return response.json()
+    def get_plugins(self, plugin_type: Optional[str] = ""):
```

Comment:
If it is optional I'd say the default should be `None`

Reply:
And perhaps we should force this to be a keyword argument as `self.get_plugins("normalizer")` reads a bit weird.